templates/OperationRequestBodySnippet.stencil

```diff
@@ -0,0 +1,6 @@
+let requestBodyjsonData = try! JSONEncoder().encode( {{ op.request.objectName }})
+let requestBodyData: Data? =  String(data: requestBodyjsonData, encoding: .utf8)!.data(using: .utf8)
```

Comment:
Why the switch from Data -> String -> Data? Suggest removing or adding a comment to clarify

Reply:
Same issue as above. Don't use `!` in production code because it's a hard crash with "unwrapped nil value". Also, I feel like if it should error out when it can't encode the object, then it should also error out if the conversion to and from String actually yields nil. In other words, I feel like it should look something like:

```Swift
guard let requestBodyData = String(...).data(using: .utf8) else {
   // some error here
}
```

Again, not that you don't need to specify the data type on the left of the equals since it can be inferred (though it did help me spot this comment so maybe that's not a bad idea!).